packages/patternfly-4/react-core/src/experimental/components/Divider/Divider.md

```diff
@@ -0,0 +1,43 @@
+---
+title: 'Divider'
+cssPrefix: 'pf-c-divider'
+typescript: true 
+propComponents: ['Divider']
+section: 'experimental'
+stage: 'early'
+---
+import { Divider } from '@patternfly/react-core/dist/esm/experimental';
+
+## Divider using hr (default)
+```js
+import React from 'react';
+import { Divider } from '@patternfly/react-core/dist/esm/experimental';
+
+DividerHr = (
```

Comment:
Did you mean for this to be a function? e.g.
```suggestion
DividerHr = () => (
```
same comment for other examples.